ner.cfg

```diff
@@ -10,7 +10,7 @@ seed = 0
 
 [nlp]
 lang = "en"
-pipeline = ["tok2vec","tagger","parser","attribute_ruler","lemmatizer","ner"]
```

Comment:
What was the rationale for taking these components out? I left them there initially in case we needed them down the road. Just wondering what's the downside of having them there.